test/util/assertTabOrder.js

```diff
@@ -3,17 +3,13 @@
 const { By, Key } = require('selenium-webdriver');
 const assert = require('assert');
 
-const getFocusedElement = async function (t) {
-  return t.context.session.executeScript(function () {
-    return document.activeElement;
-  });
-};
-
 const focusMatchesElement = async function (t, selector) {
-  return t.context.session.executeScript(function () {
-    selector = arguments[0];
-    return document.activeElement === document.querySelector(selector);
-  }, selector);
+  return t.context.session.wait(async function () {
+    return t.context.session.executeScript(function () {
+      selector = arguments[0];
+      return document.activeElement === document.querySelector(selector);
+    }, selector);
+  }, 200);
```

Comment:
Is there a reason this timeout is 200ms while the others are 500ms?

Reply:
I put the 200ms waits before I stress tests but found 500 was better -- I'll switch them all to 500 for consistency, thanks for th catch!